build-container/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM archlinux:base-devel
+
+# The following is from https://github.com/archzfs/archzfs-ci/blob/master/worker/Dockerfile
+RUN pacman -Syu --noconfirm --needed python-pipx python-twisted python-future git wget systemd-sysvcompat openresolv vi
+
+# add buildbot user and give passwordless sudo access (needed for archzfs build scripts)
+RUN groupadd -r buildbot && \
+    useradd -m -g buildbot buildbot && \
+    mkdir /worker && \
+    chown buildbot:buildbot /worker && \
+    echo "buildbot ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+# aur prep
+RUN useradd aur && \
+    echo "aur ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+# clean-chroot-manager
+USER aur
+RUN mkdir -p /tmp/ccm-install            && \
+    cd /tmp/ccm-install                  && \
+    wget https://aur.archlinux.org/cgit/aur.git/snapshot/clean-chroot-manager.tar.gz && \
+    tar -xvf clean-chroot-manager.tar.gz && \
+    cd clean-chroot-manager              && \
+    makepkg -si --noconfirm              && \
+    cd /tmp                              && \
+    rm -rfv /tmp/ccm-install
+
+USER buildbot
```

Comment:
Seeing that all commands below use `sudo`, couldn't we just switch to `root`?

Reply:
I will try that, but I wonder if some tooling is dependant on being run via sudo (such that it can detect the low-privileged username etc).
We'll see :3

Reply:
It's just for the three commands in the RUN block below. Only `ccm64` might care, I guess, and most likely not. The final `USER buildbot` at the end of the file should stay, of course.